lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -57,16 +56,21 @@ class LoadFastEnough4Pwa extends Audit {
         // Ignore requests that don't have timing data or resources that have
         // previously been requested and are coming from the cache.
         if (!record._timing || record._fromDiskCache || record._fromMemoryCache) {
-          return undefined;
+          return {url: record._url};
```

Comment:
I can use more vars in the code if it'll make that part more clear, but I'm thinking we should start being diligent about [sliming down audit results](https://github.com/GoogleChrome/lighthouse/issues/1999). Extra bits are nice, but if we're not going to use them in a renderer, they're extra weight.